packages/frontend/src/components/Explorer/ExploreSideBar/index.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+    Button,
+    Divider,
+    H3,
+    InputGroup,
+    Menu,
+    MenuDivider,
+    MenuItem,
+    NonIdealState,
+    Text,
+} from '@blueprintjs/core';
+import Fuse from 'fuse.js';
+import React, { useMemo, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { useToggle } from 'react-use';
+import { useExplores } from '../../../hooks/useExplores';
+import { useApp } from '../../../providers/AppProvider';
+import { useExplorer } from '../../../providers/ExplorerProvider';
+import { TrackSection } from '../../../providers/TrackingProvider';
+import { SectionName } from '../../../types/Events';
+import AboutFooter from '../../AboutFooter';
+import { ExploreMenuItem } from '../../ExploreMenuItem';
+import { ShowErrorsButton } from '../../ShowErrorsButton';
+import ExplorePanel from '../ExplorePanel/index';
+import {
+    FooterWrapper,
+    MenuWrapper,
+    SearchWrapper,
+    SideBarDescription,
+    SideBarTitleWrapper,
+    SwitchFilter,
+} from './ExploreSideBar.styles';
+
+const SideBarLoadingState = () => (
+    <Menu large style={{ flex: 1 }}>
+        {[0, 1, 2, 3, 4].map((idx) => (
+            <React.Fragment key={idx}>
+                <MenuItem className="bp3-skeleton" text="Hello" />
+                <MenuDivider />
+            </React.Fragment>
+        ))}
+    </Menu>
+);
+const BasePanel = () => {
+    const history = useHistory();
+    const { projectUuid } = useParams<{ projectUuid: string }>();
+    const { errorLogs } = useApp();
+    const [search, setSearch] = useState<string>('');
+    const [filterExplores, toggleFilterExplores] = useToggle(true);
+    const exploresResult = useExplores(filterExplores);
+
+    const filteredTables = useMemo(() => {
+        if (exploresResult.data) {
+            if (search !== '') {
+                return new Fuse(Object.values(exploresResult.data), {
+                    keys: ['name', 'description'],
```

Comment:
@nathaliabj we should remove `description` - we had feedback before that it made search quite poor. Alternatively we could weight the name much higher than the description. [docs on weighted search](https://fusejs.io/examples.html#weighted-search)

```
const fuse = new Fuse(books, {
  keys: [
    'description', // will be assigned a `weight` of 1
    {
      name: 'name',
      weight: 5
    }
  ]
})
```